open_humans/admin.py

```diff
@@ -1,7 +1,12 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
 
-from .models import Member, User
+from .models import Member, User, GrantProject
 
 admin.site.register(Member)
 admin.site.register(User, UserAdmin)
+
+class GrantProjectAdmin(admin.ModelAdmin):
+    list_display = [field.name for field in GrantProject._meta.get_fields()]
```

Comment:
Out of curiosity, how is this different than the default Django model admin?

Reply:
There only difference is that this allows us to display the details in a list in the admin interface without clicking through to each item:

With this ModelAdmin addition (`id` clicks through):
![](https://cl.ly/1Z3f1D1P1g3r/Screen%20Shot%202018-05-16%20at%204.07.45%20PM.png)

Without (the name clicks through):
![](https://cl.ly/1w3y2z3Y103w/Screen%20Shot%202018-05-16%20at%204.07.26%20PM.png)

Happy to remove it if we prefer to leave the admin vanilla/default!

Reply:
I like the improved version, please keep it!